src/Microsoft.AspNet.Mvc.Core/MvcOptions.cs

```diff
@@ -22,16 +22,16 @@ public class MvcOptions
         public MvcOptions()
         {
             Conventions = new List<IApplicationModelConvention>();
-            ModelBinders = new List<ModelBinderDescriptor>();
+            ModelBinders = new List<IModelBinder>();
             ViewEngines = new List<ViewEngineDescriptor>();
```

Comment:
Why do you have to call GetRequiredService? https://github.com/aspnet/Mvc/blob/dev/src/Microsoft.AspNet.Mvc/MvcOptionsSetup.cs should take in the dependencies it needs to new up the razor view engine right?